internal/ir/xds.go

```diff
@@ -172,6 +172,7 @@ type HTTPListener struct {
 	// Routes associated with HTTP traffic to the service.
 	Routes []*HTTPRoute `json:"routes,omitempty" yaml:"routes,omitempty"`
 	// IsHTTP2 is set if the upstream client as well as the downstream server are configured to serve HTTP2 traffic.
+	// TODO: review the usage about this field
```

Comment:
we need some knob at the listener level to decide if we want to enable grpc-web & grpc-stats or not.
We should edit the description of this field to say `IsHTTP2 is set if the the listener is configured to serve HTTP2 traffic."